src/Relation/Unary/Properties.agda

```diff
@@ -198,6 +199,27 @@ U-Universal = λ _ → _
 ≐′⇒≐ : Binary._⇒_ {A = Pred A ℓ₁} {B = Pred A ℓ₂} _≐′_ _≐_
 ≐′⇒≐ = Product.map ⊆′⇒⊆ ⊆′⇒⊆
 
+------------------------------------------------------------------------
+-- Between/Disjoint properties
+
+≬-symmetric : Sym {A = Pred A ℓ₁} {B = Pred A ℓ₂} _≬_ _≬_
+≬-symmetric = Product.map₂ swap
+
+⊥-symmetric : Sym {A = Pred A ℓ₁} {B = Pred A ℓ₂} _⊥_ _⊥_
+⊥-symmetric = _∘ swap
+
+≬-sym : Symmetric {A = Pred A ℓ₁} _≬_
+≬-sym = ≬-symmetric
+
+⊥-sym : Symmetric {A = Pred A ℓ₁} _⊥_
+⊥-sym = ⊥-symmetric
```

Comment:
Why do we want two names for the same proofs?

Reply:
Maybe I've misunderstood, but the `-symmetric` forms are more level polymorphic than the `-sym` ones?